TestFailure fails

From #889 and @yifan-gu found

The failure is

```
=== RUN TestFailure
--- FAIL: TestFailure (4.26s)
    rkt_exit_test.go:74: Missing hello
```

I can't reproduce on my machine. But I think it smells like a race bug in the gexpect library, which if child.Expect runs before the child process actually starts, it will error: read /dev/ptmx: input/output error, since the gexpect uses pty for starting the chile process asynchronously [1] [2]

[1] https://github.com/ThomasRooney/gexpect/blob/master/gexpect.go#L335
[2] https://github.com/kr/pty/blob/67e2db24c831afa6c64fc17b4a143390674365ef/run.go#L22

--- a/tests/rkt_exit_test.go
+++ b/tests/rkt_exit_test.go
@@ -69,7 +69,7 @@ func TestFailure(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Missing hello: %v", err)
 	}
-	err = child.Expect("main process exited, code=exited, status=20")
+	err = child.Expect("process exited, code=exited, status=20")
 	if err != nil {
 		t.Fatalf("Missing exit status: %v", err)
 	}

